WalletWasabi.Gui/Controls/WalletExplorer/CoinInfoTabViewModel.cs

```diff
@@ -0,0 +1,15 @@
+using WalletWasabi.Gui.ViewModels;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class CoinInfoTabViewModel : WasabiDocumentTabViewModel
+	{
+		public CoinInfoTabViewModel(CoinViewModel coin) : base(string.Empty)
+		{
+			Coin = coin;
+			Title = $"{coin.TransactionId[0..10]}'s Details";
```

Comment:
I would like to open a small discussion here. This means nothing to me and hard to identify the coin. How about adding the amount there? Or some kind of combination of those.

Reply:
It will be hard to put something useful in the title. I suggested @jmacato to add the `index` to the title, otherwise all my tabs have the same title, however even when the combination of txid and index is the real identifier for a coin it is also true that it mean nothing for the user. These are a few ideas:

* use txid[..4] + index    ## ex: af54/1's details
* use the amount          ## ex: 0.00873148's details
* use a combination     ## ex: 0.00873148 af54/1's details
* use the label              ## ex: zkSNACKS's details
* use an tab cuonter     ## ex: coin1, coin2 and so on.

All suck. The use of the label could make sense from the user point of view but it still sucks.

Reply:
How about the amount and the latest added Cluster. 

- Coin: 0.00873148 - zkSNACKS's details

Reply:
It can be, but I think it is too much. If I am force to take one I would go with `0.00873148 - zkSNACKS` or even shorter `0.008731 zkSNACKS`

Reply:
Yeah honestly this was just a placeholder until we can agree on what should we put in the tab's title. I think something like `0.008731 (zKSNACKs...(truncated label))` would be okay too

Reply:
How about value and anonset shield? the cluster will be very long, especially for long chains of children.

Reply:
@MaxHillebrand  Hmm... we can't use any icons on tabs, just text im afraid

Reply:
Do this: 0.008731 zkSNACKS

Reply:
@molnard should we truncate the label? i dont think we'd want a tab with a too long of a title

Reply:
We use amount on the Dequeue notification with this pattern: "Coin (0.1) Dequeued" so we apply the principle of consistency here and we use the same. If you disagree please create a new PR where we can discuss it.